python/test/unit/fem/test_dofmap.py

```diff
@@ -272,9 +272,9 @@ def test_higher_order_coordinate_map(points, celltype, order):
 
 
 @pytest.mark.skip_in_parallel
-@pytest.mark.parametrize("order", [1, 2, 3])
+@pytest.mark.parametrize("order", [1, 2])
```

Comment:
Should we instead investigate why third order fails?

Reply:
It's simply not programmed to cover the P=3 case.